packages/mobile/src/navigation/drawer/drawer.navigation.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from 'react';
+import { Dimensions } from 'react-native';
+import {
+  createDrawerNavigator,
+  DrawerNavigationOptions,
+} from '@react-navigation/drawer';
+import { RootScreenName } from '~/common/enums/enums';
+import { MainNavigation } from '../tabs/tabs.navigation';
+
+const HomeWithMenuNavigation: FC = () => {
+  const Drawer = createDrawerNavigator();
+
+  const options: DrawerNavigationOptions = {
+    headerShown: false,
+    swipeEdgeWidth: 200,
+    drawerStyle: {
+      width: Dimensions.get('screen').width,
```

Comment:
Not necessary. Best practice is to leave some space

Reply:
fixed